rocky/assets/src/bundles/app/css/themes/soft/soft.scss

```diff
@@ -24,7 +24,7 @@
 @import "manon/button-ghost"; /* Ghost button */
 @import "manon/button-icon"; /* Icon button */
 @import "manon/filter";
-@import "manon/layout";
+@import "manon/layout-set";
```

Comment:
If I understand this correctly manon/layout.scss is not used anymore and the file can be deleted?

Reply:
Yes that is true. But for the moment I kept the file as a reference until the transition between the two is complete. I will change the status on this pr from draft to ready for review once it's ready to be checked.